tests/p2p/test_protocol.py

```diff
@@ -442,11 +436,11 @@ def test_get_data(self):
             'last_block_hash': missing_tx,
             'start_from': [self._settings.GENESIS_BLOCK_HASH.hex()]
         }
-        yield self._send_cmd(self.conn.proto1, 'GET-TRANSACTIONS-BFS', json_dumps(payload))
```

Comment:
Same here.

Reply:
The `_send_cmd()` method does not return a `Deferred`, so the `yield` is not necessary.

Reply:
Ok. Thanks!